identity/identity.go

```diff
@@ -23,7 +24,7 @@ func HandlerForHTTPClient(cli *clientsidentity.Client) func(http.Handler) http.H
 			ctx := req.Context()
 			log.Event(ctx, "executing identity check middleware")
 
-			florenceToken := getFlorenceTokenFromRequest(ctx, req)
+			florenceToken := getFlorenceToken(ctx, req)
 			serviceAuthToken := req.Header.Get(common.AuthHeaderKey)
```

Comment:
As discussed, earlier, this could also be changed from `common.` to the newer `headers.` style.